src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25115.cs

```diff
@@ -0,0 +1,25 @@
+﻿#if WINDOWS // Chevron icon color is only applicable to Windows
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue25115 : _IssuesUITest
+{
+	public override string Issue => "[Windows] The color was not applied properly to the Tab Chevron icon";
+
+	public Issue25115(TestDevice device)
+	: base(device)
+	{ }
+
+	[Test]
+	[Category(UITestCategories.Shell)]
+	public void VerifyTabChevronIconColor()
+	{
+		App.WaitForElement("label");
+
+		VerifyScreenshot();
```

Comment:
Pending Windows snapshot.

Reply:
Already available in the latest build:
![image](https://github.com/user-attachments/assets/b2a62cf1-9171-4d23-a1ef-2fa2112b2850)

Reply:
I have added it.